src/gretel_trainer/relational/core.py

```diff
@@ -19,9 +24,7 @@ class MultiTableException(Exception):
 @dataclass
 class TableEvaluation:
     cross_table_report_json: Optional[Dict] = field(default=None, repr=False)
-    cross_table_sqs: Optional[int] = None
```

Comment:
@mikeknep  What are your thoughts on this? I imagine we'd want to keep the sqs properties in, since they allow the scores to be viewed via evaluations.

Reply:
The custom `__repr__` method that Tyler defined on this class (below) will ensure that we still see useful info in a notebook/repl when returning/printing `multitable.evaluations`.